DragonLoopApp/DragonLoopApp/ViewModels/MapViewModel.cs

```diff
@@ -105,6 +105,7 @@ private void RenderMapOverlay()
                     Label = stop.Name,
                     Type = PinType.Place
                 };
+                Map.CustomPins.Add(pin);

```

Comment:
Do we need to add to CustomPins and Pins? I think I noticed when building it for Android that they needed to get added to Pins to show up on the map correctly. If we need to add to both to make it work on both platforms that is fine though, but you should add a Map.CustomPins.Clear() call at the top of this method so removing pins works.